sandbox/src/routes/patient/patch-patient.js

```diff
@@ -79,6 +79,22 @@ module.exports = [
                 }
             }
 
+            // Verify that the address id is not present when required in the request
+            const invalidPatch = requestValidator.verifyAddressIdNotPresentWhenRequired(request, patientToUpdate)
```

Comment:
Rather than returning some opaque error object, just have the `verifyAddressIdNotPresentWhenRequired` throw the exceptions directly

Reply:
Changes have been made as recommended. Please review.